potatonetwork/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "server_name": "potatonetwork",
+  "nice_name": "Potato Network",
+  "direct_ip": "play.potatonetwork.de",
+  "server_wildcards": [
+    "%.potatonetwork.de"
+  ],
+  "social": {
+    "twitter": "PotatoNetworkMC",
+    "discord": "https://discord.gg/JFwVkZ7"
+  },
+  "gamemodes": {
+    "buildffa": {
+      "name": "BuildFFA"
+    },
+    "harcoreffa": {
+      "name": "HardcoreFFA"
+    },
+    "bedwars": {
+      "name": "BedWars"
+    },
+    "stickduel": {
+      "name": "StickDuel"
+    }
+  },
+  "discord": {
+    "server_id": 643165107886751757,
+    "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

Reply:
Ok, i will change it

Reply:
Should be changed